cms/static/js/views/manage_users_and_roles.js

```diff
@@ -140,7 +139,9 @@ define(['jquery', 'underscore', 'gettext', "js/views/baseview",
                     roles = _.object(_.pluck(this.roles, 'key'), _.pluck(this.roles, "name")),
                     adminRoleCount = this.getAdminRoleCount(),
                     viewHelpers = {
-                        format: function (template, data) { return _.template(template, data, {interpolate: /\{(.+?)}/g}); }
+                        format: function (template, data) {
```

Comment:
Here I got rid of _.template in favor of interpolate. I could revert this change though now that I understand how to pass settings to _.template with the new syntax.